server/protocols/jmap/src/test/java/org/apache/james/jmap/model/UpdateMessagePatchTest.java

```diff
@@ -80,7 +86,116 @@ public void applyStateShouldReturnSeenWhenPatchSetsSeenOnSeenMessage() {
     }
 
     @Test
-    public void isIdentityShouldReturnTrueWhenNoFlags() {
+    public void applyStateShouldThrowWhenBothIsFlagAndKeywords() {
```

Comment:
Does it mean I can build an object with an invalid state I can not work with? What is then the value of the builder pattern?